dependencies/config/toon.dc

```diff
@@ -1,3946 +0,0 @@
-from direct.distributed import DistributedObject/AI/UD
```

Comment:
This must be a mistake. Can you please reupload this file?

Reply:
Try the source yourself. You will find that this is not a mistake at all.